server/src/com/cloud/network/StorageNetworkManagerImpl.java

```diff
@@ -223,7 +225,7 @@ public StorageNetworkIpRange createIpRange(final CreateStorageNetworkIpRangeCmd
         checkOverlapPrivateIpRange(podId, startIp, endIp);
         checkOverlapStorageIpRange(podId, startIp, endIp);
 
-        StorageNetworkIpRangeVO range = null;
+        //StorageNetworkIpRangeVO range = null;
```

Comment:
I know :) Still busy testing, please read the last comment of my PR message. I PR'd the first commit, then pushed the second one for testing.